pyproject.toml

```diff
@@ -1,19 +1,97 @@
+[tool.poetry]
+name = "epyq"
+version = "0.0.0"
+description = ""
+authors = ["alex anker <alex.anker@epcpower.com>"]
+license = "GPL-2.0-or-later"
+
+[tool.poetry-dynamic-versioning]
+enable = true
```

Comment:
I see this set as `enable = false` elsewhere. Is this correct?

Reply:
I think i had it working for dev setups. So poetry-dynamic-versioning would trigger for some poetry commands.  Ill set it to false though to be less confusing.

Reply:
I think i had it working for dev setups. So poetry-dynamic-versioning would trigger for some poetry commands.  Ill set it to false though to be less confusing.